docs/man/git-lfs-ext.1.ronn

```diff
@@ -0,0 +1,25 @@
+git-lfs-ext(1) - View extension details
+==============================================================
+
+## SYNOPSIS
+
+`git lfs ext list` [<name>...]
+
+## DESCRIPTION
+
+Git LFS extensions enable the manipulation of files streams
+during smudge and clean.
+
+## EXAMPLES
+
+* List details for all extensions
+
+    `git lfs ext list`
+
+* List details for the specified extensions
+
+    `git lfs ext list 'foo' 'bar'`
+
```

Comment:
Yeah thinking about it more today -- I'd only want `show` if the command had more output. Right now it's just a list of extension names.